packages/mdc-textfield/_mixins.scss

```diff
@@ -41,24 +41,24 @@
   }
 }
 
-@mixin mdc-text-field-invalid-label-shake-keyframes_($modifier, $positionY) {
+@mixin mdc-text-field-invalid-label-shake-keyframes_($modifier, $positionY, $scale:.75) {
   @keyframes invalid-shake-float-above-#{$modifier} {
     0% {
-      transform: translateX(0) translateY(-#{$positionY}) scale(.75, .75);
+      transform: translateX(0) translateY(-#{$positionY}) scale(#{$scale}, #{$scale});
```

Comment:
It should be fine to only specify one scale parameter, since `scale` defaults to using the same number for both dimensions if only one is specified. (See also the instances of this in mdc-text-field.scss)